testbed/testcase.py

```diff
@@ -16,5 +16,13 @@ def assertIsChildOf(self, spanA, spanB):
     def assertIsNotChildOf(self, spanA, spanB):
         return self.assertNotEqual(spanA.parent_id, spanB.context.span_id)
 
+    def assertHasNoParent(self, span):
+        return self.assertIsNone(span.parent_id)
+
     def assertNamesEqual(self, spans, names):
         self.assertEqual(list(map(lambda x: x.operation_name, spans)), names)
+
+    def assertEmptySpan(self, span, name):
```

Comment:
I'd leave the name assertion out of this test case ;)

Reply:
It's super convenient to assert span with name, because every span must has name :) I can make `name` optional in this helper.